pxemgr/pxemanager.go

```diff
@@ -195,6 +195,9 @@ func PXEManager(c PXEManagerConfiguration, cluster *hostmgr.Cluster) (*pxeManage
 		mgr.etcdDiscoveryUrl = mgr.config.TemplatesEnv["mayu_https_endpoint"].(string) + "/etcd"
 	}
 
+	// we need to do this on boot time to ensure all newly added Network.ExtraNICs have properly assigned IP to all hosts
+	mgr.checkAdditionalNICAddresses()
+
```

Comment:
this is an important part of code which on boot checks if proper IPs are assigned to all Extra NICs as there can be scenario when customer want to add new NICs to already installed cluster

implementation is lower in the diff